sample/src/main/res/layout/sample_activity.xml

```diff
@@ -59,7 +59,7 @@
         </android.support.design.widget.TabLayout>
```

Comment:
I think that the `android.support` should not be used anywhere. Do a `android.support` search on the sample module from the IDE to view any unmigrated support library usages

Reply:
Yeah, I missed these. I've found remaining references and updated them :+1: